apps/studio/src/features/editing-experience/components/TipTapProseComponent.tsx

```diff
@@ -57,7 +57,7 @@ function TipTapProseComponent({ content }: TipTapComponentProps) {
     },
   })
 
-  const updatePageState = (editorContent: JSONContent) => {
+  const updatePageState = (editorContent: JSONContent | undefined) => {
```

Comment:
question - this is required so that it becomes `undefined` and then we prevent the users from saving rgiht?

Reply:
yup